common/config/config.go

```diff
@@ -63,7 +63,7 @@ type (
 		// Blobstore is the config for setting up blobstore
 		Blobstore Blobstore `yaml:"blobstore"`
 		// Authorization is the config for setting up authorization
-		Authorization Authorization `yaml:authorization`
+		Authorization Authorization `yaml:"authorization""`
```

Comment:
Seems like there is one more double quote at the end. Probably this cause the config doesn’t work